internal/ingest/linksystem.go

```diff
@@ -274,10 +274,15 @@ func (ing *Ingester) ingestAd(publisherID peer.ID, adCid cid.Cid, ad schema.Adve
 		//
 		// Use string search until then.
 		wrappedErr := fmt.Errorf("failed to sync first entry while checking entries type: %w", err)
-		if strings.Contains(err.Error(), "content not found") {
+		msg := err.Error()
+		switch {
+		case
+			strings.Contains(msg, "content not found"),
```

Comment:
I wonder if there is a better way to distinguish between graph sync errors, so that it won't break once an error message changes?

Reply:
if the error is not wrapped that's the best we can do. See comment above. Having said that I am not sure if `SkipMe` is wrapped or not.